app/views/shared/forms/_types_of_income_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with(
+      model: @legal_aid_application,
+      url: form_path,
+      method: :patch,
+      local: true
+    ) do |form| %>
+
+  <%= govuk_form_group show_error_if: @legal_aid_application.errors.present? do %>
+    <%= govuk_fieldset_header do %>
+      <h1 class="govuk-fieldset__heading govuk-!-padding-bottom-2"><%= page_title %></h1>
+      <h2 class="govuk-heading-m"><%= t('.select_all_that_apply') %></h2>
+    <% end %>
+
+    <div class="deselect-group" data-deselect-ctrl="#none_selected">
+      <%= transaction_type_check_boxes(
+            form: form,
+            transaction_types: TransactionType.credits,
+            hints: {
+              friends_or_family: t('.financial_help_examples')
+            }
+          ) %>
+    </div>
+
+    <p class="govuk-!-padding-top-4"><%= t('.or_break') %></p>
+
+    <div class="govuk-checkboxes">
+      <div class="govuk-checkboxes__item">
+        <%= check_box_tag :none_selected, 'true', false, class: 'govuk-checkboxes__input' %>
+        <%= label_tag :none_selected, t('.none_selected'), class: 'govuk-label govuk-checkboxes__label' %>
+      </div>
+    </div>
+
+  <% end %>
+  <%= next_action_buttons(form: form) %>
```

Comment:
the provider page should show the draft button

Reply:
Good point! Thank you.